iothub/device/samples/getting started/RoutingTutorial/SimulatedDevice/Program.cs

```diff
@@ -135,15 +139,11 @@ private static async Task SendDeviceToCloudMessagesAsync(DeviceClient deviceClie
                     pointInfo = infoString
                 };
                 // serialize the telemetry data and convert it to JSON.
-                string telemetryDataString = JsonConvert.SerializeObject(telemetryDataPoint);
+                string telemetryDataString = JsonSerializer.Serialize(telemetryDataPoint);
 
                 // Encode the serialized object using UTF-8 so it can be parsed by IoT Hub when
                 // processing messaging rules.
-                using var message = new Message(Encoding.UTF8.GetBytes(telemetryDataString))
-                {
-                    ContentEncoding = "utf-8",
-                    ContentType = "application/json",
-                };
+                var message = new OutgoingMessage(Encoding.UTF8.GetBytes(telemetryDataString));
```

Comment:
New way to do this in v2. Don't need to serialize or encode. It'll be done for you.

```csharp
var message = new OutgoingMessage(telemetryDataPoint);
```